PYME/recipes/processing.py

```diff
@@ -2848,3 +2848,36 @@ def run(self, input_name):
         series_epers.mdh['Camera.ADOffset'] = 0
 
         return series_epers
+    
+
+@register_module('Rotate180')
+class Rotate180(Filter):
+    """
+    Rotate the image 180 degrees
+
+    Parameters
+    ----------
+    input_name : PYME.IO.ImageStack
+    output_name : PYME.IO.ImageStack
+    """
+    def applyFilter(self, data, chanNum, i, image0):
+        return data[::-1,::-1]
+    
+
+@register_module('SplineFitBgSub')
```

Comment:
```suggestion
@register_module('SplineFitBackgroundSubtract')
```

Module names should be as descriptive as possible to increase the chance that people will hit on them when searching in the recipe GUI.